index.html

```diff
@@ -592,11 +592,11 @@ <h3>
         </p>
         <p data-dfn-for="PerformanceResourceTiming">
           The <dfn>nextHopProtocol</dfn> getter steps are to [=/isomorphic
-          decode=] <a>this</a>'s <a data-for="PerformanceResourceTiming">timing
-          info</a>'s [=fetch timing info/final connection timing info=]'s
-          [=connection timing info/ALPN negotiated protocol=]. See
-          <a data-cite="FETCH#record-connection-timing-info">Recording
-          connection timing info</a> for more info.
+          decode=] <a>this</a>'s [=PerformanceResourceTiming/timing info=]'s
+          [=fetch timing info/final connection timing info=]'s [=connection
+          timing info/ALPN negotiated protocol=]. See <a data-cite=
```

Comment:
Can we also drop the data-cite here?

Reply:
Possibly, but it's explicitly marked as a "private" definition (noexport) by Fetch. @annevk, not sure if you want to export that? Do you have a suggestion of what we could link to?

Reply:
OK, let's not block on it then. We can open a separate issue for this